### DISCUSSION TITLE
Unable to create new repository

### DISCUSSION THREAD

**Discussion by kammar20** [community/community#30808] [Other Feature Feedback, Questions, & Ideas]
I am having some difficulties creating a new repository. The "Create Repository" button is disabled. However, I am able to create a repo in incognito mode.

**Comment by airtower-luna** [comment]
In many browsers incognito mode changes which add-ons are enabled. If you use any add-ons, check the difference, and see if any add-on that doesn't run in incognito mode causes the problem. The easiest way to do that is disabling each of them in turn. Then check the settings for that add-on, and possibly make a bug report.

**Comment by jfullstackdev** [comment]
to add, if you know how to use Chrome Profile, that will clear you of those extensions, say, a Chrome for GitHub only, for your work related tasks, so that installations from the other profile will not affect the others

**Comment by Neozxc** [comment]
Would suggest turning off every tab in your browser and then turning off browser and re-opening again. You can also try clearing cookies and things like cache. Never got this error, interesting.

**Reply by kammar20** [reply]
Clearing cookies works, thanks😊
